### PR TITLE
Canary stage now allows configuration of the TERMINATE action

### PR DESCRIPTION
**Pull request by ajordens**

### CODE REVIEW

**Comment by ajordens** [comment]
@anotherchrisberry - I have no idea if the getterSetter thing is good or not, I didn't have anything concrete in the model to bind to.

Also, I would have written a test case but have no idea how to skeleton up a default spec.

File: app/scripts/modules/netflix/pipeline/stage/canary/canaryStage.js [hunk 1]
```diff
@@ -45,6 +45,40 @@ module.exports = angular.module('spinnaker.netflix.pipeline.stage.canaryStage',
     $scope.stage.canary.canaryConfig.canaryAnalysisConfig = $scope.stage.canary.canaryConfig.canaryAnalysisConfig || {};
     $scope.stage.canary.canaryConfig.canaryAnalysisConfig.notificationHours = $scope.stage.canary.canaryConfig.canaryAnalysisConfig.notificationHours || [];
 
+    $scope.stage.canary.canaryConfig.actionsForUnhealthyCanary = $scope.stage.canary.canaryConfig.actionsForUnhealthyCanary || [
+      {action: 'DISABLE'},
+      {action: 'TERMINATE', delayBeforeActionInMins: 60}
+    ];
+
+    this.terminateUnhealthyCanaryEnabled = function (isEnabled) {
+      if (isEnabled === true) {
```
Comment (anotherchrisberry) [inline app/scripts/modules/netflix/pipeline/stage/canary/canaryStage.js]: You can just do:
```javascript
if (isEnabled) {
```

File: app/scripts/modules/netflix/pipeline/stage/canary/canaryStage.js [hunk 2]
```diff
@@ -45,6 +45,40 @@ module.exports = angular.module('spinnaker.netflix.pipeline.stage.canaryStage',
...
+        $scope.stage.canary.canaryConfig.actionsForUnhealthyCanary = [
+          {action: 'DISABLE'},
+          {action: 'TERMINATE', delayBeforeActionInMins: 60}
+        ];
+      } else if (isEnabled === false) {
```
Comment (anotherchrisberry) [inline app/scripts/modules/netflix/pipeline/stage/canary/canaryStage.js]: Can be simplified to:
```javascript
} else {
```

**Comment by anotherchrisberry** [comment]
LGTM

**Pull request merged**